packages/cdktf-cli/lib/get/generator/provider-schema.ts

```diff
@@ -75,34 +75,36 @@ interface ModuleIndex {
 const transformVariables = (variables: any) => {
   const result = []
 
-  for (const name of Object.keys(variables)) {
-    const variable = variables[name][0]
-    let variableType: string;
-    // eslint-disable-next-line no-prototype-builtins
-    if (variable.hasOwnProperty('type') == false && variable.hasOwnProperty('default') == true) {
-      switch (typeof variable['default']) {
-        case "boolean": variableType = 'bool' ; break;
-        case "number": variableType = 'number' ; break;
-        default: variableType = 'any';
+  if (variables) {
```

Comment:
I think a guard clause would be nicer here, something like `if (!variables) return result;`  Or are there other changes than this if clause?

Reply:
That was the only change. I switched to a guard, though that did require being more explicit about type.

Reply:
Cool - thanks 👍